programmers/k-th-number.js

```diff
@@ -0,0 +1,15 @@
+function solution(arry, commands){
+    const cd_len = commands.length;
+    let results = [];
```

Comment:
배열은 const로 선언해주세요
배열이라 복수형으로 변수이름짓는건 너무 좋아요
```suggestion
    const results = [];
```